packages/terra-docs/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "terra-docs",
+  "main": "lib/Docs.js",
+  "version": "0.0.0",
+  "private": true,
+  "description": "{insert description}",
```

Comment:
add description

Reply:
coming soon, when content is actually added.